src/harness/testcases/WINNF_FT_S_MCP_testcase.py

```diff
@@ -449,20 +567,28 @@ def executeSingleMCPIteration(self, test_type, iteration_content, sas_test_harne
 
     # Register individual Cbsds(Domain Proxy with single CBSD)
     for cbsd_record in iteration_content['cbsdRecords']:
-      proxy = test_harness_objects.DomainProxy(cbsd_record['clientCert'], cbsd_record['clientKey'], self)
-      proxy.registerCbsdsAndRequestGrants([cbsd_record['registrationRequest']], [cbsd_record['grantRequest']])
+      proxy = test_harness_objects.DomainProxy(self, cbsd_record['clientCert'], cbsd_record['clientKey'])
+      proxy.registerCbsdsAndRequestGrants([cbsd_record['registration']], [cbsd_record['grants']])
       domain_proxy_objects.append(proxy)
 
     # Step 11 : Send heartbeat request managed by SAS UUT
     for domain_proxy_object in domain_proxy_objects:
       domain_proxy_object.heartbeatForAllActiveGrants()
       cbsds.append(domain_proxy_object.getCbsdsWithAtLeastOneAuthorizedGrant())
 
-    # Step 12 : Invoke Aggregate Interference Model
-    # TODO: To invoke Aggregate Interference Model
+    cbsd_records = getFullActivityDumpSasTestHarness(cbsds).getCbsdRecords()
```

Comment:
The argument to `getFullActivityDumpSasTestHarness` is supposed to be a SAS interface, not `cbsds`.

Reply:
Working on this...

Reply:
I'm confused: you are supposed to pass a SAS interface to the function `getFullActivityDumpSasTestHarness()`, not a list of CBSDs or DPs. What are you trying to do here?